android-sdk/src/main/java/ai/evolv/android_sdk/EvolvStoreImpl.java

```diff
@@ -287,14 +299,98 @@ private void updateGenome(JsonArray value) {
     }
 
     private void updateConfig(JsonObject value) {
```

Comment:
This code needs unit tests -- all the logic in the PR is in this class. I need for.  this review to help understand what you're doing

Reply:
I know that the SDK logic is not very simple, but the tests will fully help you figure it out. I have already written the first "EvolvConfig" test, I will write others as well. Thank you